docs/docsite/rst/user_guide/playbooks_tests.rst

```diff
@@ -380,7 +380,80 @@ The following tasks are illustrative of the tests meant to check the status of t
 
 .. note:: From 2.1, you can also use success, failure, change, and skip so that the grammar matches, for those who need to be strict about it.
 
-
+.. _type_tests:
+
+Type Tests
+==========
+
+When looking to determine types, it may be tempting to use the ``type_debug`` filter and compare that to the string name of that type, however, you should instead use type test comparisons, like these::
```

Comment:
```suggestion
When looking to determine types, it may be tempting to use the ``type_debug`` filter and compare that to the string name of that type, however, you should instead use type test comparisons, such as:

.. code-block:: yaml

```